.dockerignore

```diff
@@ -3,5 +3,6 @@
 !files/dev-perms.pub.pem
 !files/dev-reticulum-jwk.json
 !files/dev-reticulum.conf
+!files/coturn/
```

Comment:
This runs counter to the precedent of explicitly allowing each file in the `files/` directory. I think we should either list each one or replace all the `!files/` lines with a single `!files/*` entry for consistency. What do you think?

Reply:
Not sure what's better but adding the entries sounds good.